src/benchmark/benchmark.cc

```diff
@@ -60,7 +60,8 @@ void Benchmark::Run() {
 
   defaults->Set<int>(SearchParams::kMiniBatchSizeId.GetId(), 256);
   defaults->Set<float>(SearchParams::kFpuReductionId.GetId(), 1.2f);
-  defaults->Set<float>(SearchParams::kCpuctId.GetId(), 3.4f);
+  defaults->Set<float>(SearchParams::kCpuctId.GetId(), 3.0f);
```

Comment:
Maybe make those defaults and override in selfplay?

Reply:
Why not a helper function in SearchParams that changes the defaults?